crescent/bot.py

```diff
@@ -199,7 +210,7 @@ async def on_crescent_command_error(
         print_exception(exc.__class__, exc, exc.__traceback__)
 
     async def on_crescent_event_error(
-        self, exc: Exception, event: Event, was_handled: bool
+        self, exc: Exception, event: hk_Event, was_handled: bool
```

Comment:
hk_Event feels weird too (I see why you have aio_Event though).

Maybe `HikariEvent` and `AsyncIOEvent`?

Reply:
I think the way i have it currently is more clear because `aio` and `hikari` aren't part of the class name. They're just a prefix to prevent shadowing.

Reply:
will they show up as hk_Event and aio_Event inside of the signature on hover for vscode?

Reply:
both of them show up as `Event`